docs/agent-api.asciidoc

```diff
@@ -646,3 +646,64 @@ Optionally, a type may also be provided to group lambdas together. By default,
 "lambda" will be used as the type name.
 
 Read more lambda support in the <<lambda,Lambda>> article.
+
+[[apm-add-patch]]
+==== `apm.addPatch(name, handler])`
+
+[source,js]
+----
+apm.addPatch('timers', (exports, agent, version, enabled) => {
```

Comment:
Let's just discuss the arguments that this callback is being called with. The function signature have evolved randomly over time and this hasn't been a big concern as it wasn't a public API. Now that we're committing to it, we should just talk about if this is the right API

Reply:
Well, we will _always_ need the exports, agent and version. I think `enabled` could maybe fit into a config object though. 🤔